products/sle/main.pm

```diff
@@ -655,6 +655,9 @@ if (is_jeos) {
 if (is_kernel_test()) {
     load_kernel_tests();
 }
+elsif (is_container_test) {
+    load_container_tests();
+}
```

Comment:
Please don't forget to do this change in `products/opensuse/main.pm` too as we follow the Factory First approach.

Reply:
O3 jobs don't use yaml schedule. My idea was to gradually adapt this to not break O3 jobs. First convert yaml schedules in OSD to this new way, and then maybe make O3 jobs use this function. 
Factory first applies to test coverage, I am not adding any new test or anything.

Reply:
Does it mean removal of `load_extra_tests_docker` and `main_common.pm` clean up should come as a follow-up PR?

Reply:
That could be the next thing to tackle, yes